### PR TITLE
[GUI] Capitalize First Letter in Menu Option Words

### PR DESCRIPTION
**Pull request by Rock-N-Troll**
(Recreated branch due to commit issue)

**Existing:**
image

**New:**
image

### CODE REVIEW

**Comment by CaveSpectre11** [comment]
Please squash your commits

**Comment by Rock-N-Troll** [comment]
somehow I manage to always get coat's last commit involved, but hopefully shouldn't be an issue here

File: src/qt/veil/balance.cpp [hunk 1]
```diff
@@ -129,7 +129,7 @@ void Balance::onBtnBalanceClicked(int type){
             thirdBalance = balances.basecoin_balance;
             widget = ui->btnBalance;
             posy = 0;
-            posx = widget->pos().rx()+150;
+            posx = widget->pos().rx()+190;
```
Comment (CaveSpectre11) [inline src/qt/veil/balance.cpp:132]: Make sure this is covered in your description of what was done.

**Review by CaveSpectre11 — COMMENTED** [review]
Not sure what happened here:

image

**Review by codeofalltrades — APPROVED** [review]
ACK d9720e35b070078d2c410a37feaece88699db972
Tested in Win10

**Pull request merged**